### PR TITLE
Change the init check to only consider Persist.toml

### PR DESCRIPTION
**Pull request by sahanHe**
## Purpose
$subject
Related to:https://github.com/ballerina-platform/ballerina-standard-library/issues/3663

## Checklist
- [ ] Linked to an issue
- [ ] Updated the specification
- [ ] Updated the changelog
- [x] Added tests

### CODE REVIEW

File: persist-cli/src/main/java/io/ballerina/persist/cmd/Init.java [hunk 1]
```diff
@@ -95,16 +95,21 @@ public void execute() {
...
-            generateConfigurationBalFile();
-            outStream.println("Added new Ballerina module at modules/clients");
-            outStream.println("Created database_configuration.bal file with configurations.");
+            if (!databaseConfig.exists()) {
+                generateConfigurationBalFile();
+                outStream.println("Added new Ballerina module at modules/clients");
+                outStream.println("Created database_configuration.bal file with configurations.");
+            } else {
+                generateConfigurationBalFile();
+                outStream.println("Replaced database_configuration.bal file with default configurations.");
```
[the remark this answers is not in the document]
Comment (sahanHe) [reply persist-cli/src/main/java/io/ballerina/persist/cmd/Init.java:111]: I was not sure whether it's valid to say `"Created database_configuration.bal file with configurations."` when there is already a file with the same name

[the remark this answers is not in the document]
Comment (niveathika) [reply persist-cli/src/main/java/io/ballerina/persist/cmd/Init.java:111]: For init case, 
```log
// Created Persist.toml file
// Created database_configuration.bal
```

For reinit case,
```log
// Delete `clients` module
// Created Persist.toml
// Created database_configuration.bal
// Created clients...
```

[the remark this answers is not in the document]
Comment (daneshk) [reply persist-cli/src/main/java/io/ballerina/persist/cmd/Init.java:112]: I am +1 for deleting the `clients` module and creating messages

[the remark this answers is not in the document]
Comment (sahanHe) [reply persist-cli/src/main/java/io/ballerina/persist/cmd/Init.java]: updated in [0b20cf6](https://github.com/ballerina-platform/persist-tools/pull/61/commits/0b20cf654b99888732ca99e7d9b5db2a174002a9)

**Review by niveathika — APPROVED** [review]

**Pull request merged**